template/rwa-onboarding/Goerli-DssSpell.sol

```diff
@@ -35,8 +35,11 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     }
 
     function actions() public override {
+        // ---------------------------------------------------------------------
+        // Includes changes from the DssSpellCollateralAction
         onboardNewCollaterals();
```

Comment:
Suggestions to reflect the [latest spells cast](https://github.com/makerdao/spells-mainnet/blob/master/archive/2022-10-26-DssSpell/DssSpell.sol) (1/2):

```suggestion
        // ---------------------------------------------------------------------
        // Includes changes from the DssSpellCollateralAction
        collateralAction();
```